frontend/src/components/icon.tsx

```diff
@@ -47,6 +50,8 @@ import instagram from '@iconify-icons/mdi/instagram'
 import twitter from '@iconify-icons/mdi/twitter'
 
 export const ICONS = {
+    arrowLeft,
```

Comment:
want to avoid using this `<Icon>` component and use the tabler/react-icons (eventually want to remove this component)

Reply:
I'll modify all the Icons